poplar.js

```diff
@@ -55,14 +55,14 @@ poplar.parse = function(html) {
   debug('parse', html);
 
   var formatted = html
-    .replace(/\n/g, '')
+    .replace(/\s*\n\s*/g, '')
```

Comment:
removes all the line returns _and_ potential spaces just before/after, like trailing spaces for example